lifetimes/fitters/pareto_nbd_fitter.py

```diff
@@ -147,9 +150,19 @@ def conditional_probability_alive(self, frequency, recency, T):
         x, t_x = frequency, recency
         r, alpha, s, beta = self._unload_params('r', 'alpha', 's', 'beta')
 
-        A_0 = np.exp(self._log_A_0([r, alpha, s, beta], x, t_x, T))
-        return 1. / (1. + (s / (r + s + x)) *
-                     (alpha + T) ** (r + x) * (beta + T) ** s * A_0)
+        A_0 = 0
+        if alpha >= beta:
```

Comment:
They're slightly different, for example the third argument of `hyp2f1` (see denominator). I'm going to see if I can factor or at least understand how this numerical optimization works. I found [one additional source](http://www.brucehardie.com/notes/008/pareto_nbd_MATLAB.pdf).